nodestream/schema/printers/graph_schema_extraction.py

```diff
@@ -0,0 +1,25 @@
+from .graphql_schema_printer import GraphQLSchemaPrinter
+from collections import defaultdict
+
+class Neo4jLLMGraphSchemaExtraction(GraphQLSchemaPrinter):
```

Comment:
This class should extend from the `SchemaPrinter` base class and specify an alias. See the defiinition of `GraphQLSchemaPrinter`.